Approved on 2024-08-05 - Open Source in Business (#336)
todogroup/governance: resolutions/proposed/2024-28.md

Reviewer: Rename to `resolutions/approved/2024-06-24.md`

Resolution YYYY-NN

> Status: Proposed on 2024-06-24

Reviewer: ```suggestion
Status: Accepted
Proposed date: 2024-06-24
```

> Subject: New Focuss Group - Open Source in Business Collaborative Whitepaper / Guide

Reviewer 2: Focus

Reviewer: ```suggestion
Subject: New Focus Group - Open Source in Business Collaborative Whitepaper / Guide
```

 ## Proposal
 
During the Touchpoint call on Thursday, May 16th, participants presented the idea of building a collaborative project focused on open source in business. The proposal can be found [here](https://github.com/todogroup/ospology/discussions/481).

The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments bellow [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and
agreed to define an outline to enable the group to begin creating content.

The request is for the SC to review this proposal to approve the creation of a new repository under TODO GH org to work collaboratively on the outline of this guide.

Resolved:

WHEREAS, the charter of the TODO Group, dated YYYY-MM-DD, empowers the Steering Committee to {category of action};

That the committee {approves/declines the proposed action}.
Vote Results

>     Annania Melaku - {approve/decline/abstain/absent}
>     Ashley Wolf - {approve/decline/abstain/absent}
>     Brittany Istenes - {approve/decline/abstain/absent}
>     Georg Kunz - {approve/decline/abstain/absent}
>     Leslie Hawthorn - {approve/decline/abstain/absent}
>     Nik Peters - {approve/decline/abstain/absent}
>     Stephen Augustus - {approve/decline/abstain/absent}

Reviewer: ```suggestion
    Annania Melaku - approve
    Ashley Wolf - approve
    Brittany Istenes - absent
    Georg Kunz - approve
    Leslie Hawthorn - approve
    Nik Peters - absent
    Stephen Augustus - approve
```

ref: https://github.com/todogroup/governance/pull/336#issuecomment-2268082101

> Resolution YYYY-NN

Author: ```suggestion
Resolution 2024-08
```

Status: Accepted
Proposed date: 2024-06-24
Subject: New Focus Group - Open Source in Business Collaborative Whitepaper / Guide

 ## Proposal
 
During the Touchpoint call on Thursday, May 16th, participants presented the idea of building a collaborative project focused on open source in business. The proposal can be found [here](https://github.com/todogroup/ospology/discussions/481).

> The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments bellow [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and

Reviewer: ```suggestion
The following month, during the Touchpoint call on Thursday, June 27th, the community reviewed the interest shared (see comments [here](https://github.com/todogroup/ospology/discussions/481)) and started to draft a list of challenges and
```

agreed to define an outline to enable the group to begin creating content.

> The request is for the SC to review this proposal to approve the creation of a new repository under TODO GH org to work collaboratively on the outline of this guide.

Reviewer: ```suggestion
The request is for the Steering Committee to review this proposal to approve the creation of a new repository under TODO Group GitHub organization to work collaboratively on the outline of this guide.
```

Resolved:

> WHEREAS, the charter of the TODO Group, dated YYYY-MM-DD, empowers the Steering Committee to {category of action};

Reviewer: Needs update

> WHEREAS, the charter of the TODO Group, dated YYYY-MM-DD, empowers the Steering Committee to {category of action};

Author: ```suggestion
The Steering Committee approves the creation of a new repository under TODO Group GitHub organization to work collaboratively on the outline of this guide;
```

> That the committee {approves/declines the proposed action}.

Reviewer: ```suggestion
That the committee approves the proposed action.
```

> Vote Results

Reviewer: Needs heading?

> That the committee approves the proposed action.

Author: ```suggestion
That the committee approves the creation of a new repository under TODO Group GitHub organization to work collaboratively on the outline of this guide.
```

> Vote Results

Author: ```suggestion
## Vote Results
```

    Annania Melaku - approve
    Ashley Wolf - approve
    Brittany Istenes - absent
    Georg Kunz - approve
    Leslie Hawthorn - approve
    Nik Peters - absent
    Stephen Augustus - approve

> Rationale

Reviewer: Needs update (and heading?)
